api/src/main/java/org/eclipse/microprofile/rest/client/spi/RestClientBuilderResolver.java

```diff
@@ -41,14 +38,15 @@
  * This class provides a default implementation which uses the service loader
  * pattern to look for all implementations of <code>RestClientBuilder</code> and
  * creates a new builder with the highest priority specified by the
- * {@link Priority} annotation.
+ * {@link javax.annotation.Priority} annotation.
```

Comment:
The javadoc's wrong now.

Reply:
Yes, whole paragraph starting "This class provides..." should be removed

Reply:
I think you missed this one, @johnament 
I went on and fixed the javadocs directly with the commit https://github.com/eclipse/microprofile-rest-client/commit/d8aa302594fb2d871b03407fe67b40e29ebdf11c